wp-content/themes/core/components/loop_items/search/search.php

```diff
@@ -6,26 +6,30 @@
 $c = Search_Controller::factory();
 ?>
 
-<article class="item-loop item-loop--search item-loop--<?php echo esc_attr( get_post_type() ); ?>">
+<article class="search-result search-result--<?php echo esc_attr( get_post_type() ); ?>" data-js="use-target-link">
```

Comment:
To be totally honest, I'd prefer to see a card component used here instead of a separate template item. It creates a nice abstraction of loop items being cards that could be reused in any context (not "locked" into the results template).

Codet and Durham both have their results setup that way, if you need a reference. Let me know if you want to chat it through together.

Reply:
Agreed!